apps/trading/components/rewards-container/use-payout-per-rank.ts

```diff
@@ -0,0 +1,68 @@
+import type { Maybe, RankTable } from '@vegaprotocol/types';
+
+// Use rank table to calculate payouts per rank
+// as per https://docs.google.com/spreadsheets/d/1RpWKnGEf4eYMxjI-feauRa9vWz3pNGdP05ejQrwWatg/
+export const usePayoutPerRank = (
+  rankTable: Maybe<Maybe<RankTable>[]> | undefined
```

Comment:
(non blocking): I'd prefer not to use the `Maybe` type here and would favour the more basic `rankTable: RankTable[] | undefined`. 

I think its ended up being required because for the reward types we are using the generated default type rather than the generated type from the query itself, which would be more accurate.

Reply:
yeah Maybe ended up being required that's why you see Maybe here but I'll find a way to remove it